data/survey/6001/meta.json

```diff
@@ -0,0 +1,48 @@
+[
+    {
+        "type": "TEXT",
+        "question": "What is your first name?"
+    },
+    {
+        "type": "CHOICE",
+        "question": "Are you interested in a career in the technology industry?",
+        "options": [
+            "As of now, I don't plan on it.",
+            "Not sure...",
+            "I'm open to it. :)",
+            "Absolutely!"
```

Comment:
I think rephrasing this would be helpful, maybe along the lines of "How interested are you in a career in the technology industry?" with responses "Very interested," "Somewhat interested," "Slightly interested," and "Not interested" would help have clearer answers.

Reply:
I think want a colloquial tone to reduce perceiving this course as a chore. For data collection purposes, we'd just differentiate the first two from the second two answers.